cmake/GinkgoConfig.cmake.in

```diff
@@ -133,4 +134,10 @@ include(${CMAKE_CURRENT_LIST_DIR}/hip_helpers.cmake)
 # NOTE: we do not export benchmarks, examples, tests or devel tools
 #     so `third_party` libraries are currently unneeded.
 
+# populate CUDA_HOST_COMPILER if Ginkgo was built with CUDA
+if (GINKGO_BUILD_CUDA AND GINKGO_CUDA_HOST_COMPILER AND NOT CMAKE_CUDA_HOST_COMPILER)
```

Comment:
I wasn't entirely sure whether this should also depend on GINKGO_BUILD_HIP/GINKGO_HIP_PLATFORM

Reply:
HIP with HIP_PLATFORM cuda also has its own thing (setting ccbin etc), so maybe we should also do something in that case

Reply:
But is this actually a use case we want to target? Do we expect users to include Ginkgo in their HIP code on NVIDIA machines? And would there be something we could do to help them?